pages/course/[code].tsx

```diff
@@ -40,16 +66,81 @@ const CoursePage: NextPage = () => {
               fontWeight="500"
               fontFamily="title"
             >
-              {courseData?.name}
+              {course.name}
```

Comment:
This should be `course?.name` or an empty string since some course names on the campus book store can not be retrieved from the academic calendar website.